server/sonar-server/src/main/java/org/sonar/server/permission/ws/UsersAction.java

```diff
@@ -89,20 +89,20 @@ public void handle(Request wsRequest, Response wsResponse) throws Exception {
       Optional<ComponentDto> project = dependenciesFinder.searchProject(dbSession, request);
       checkProjectAdminUserByComponentDto(userSession, project);
       PermissionQuery permissionQuery = buildPermissionQuery(request, project);
-      UsersResponse usersResponse = usersResponse(permissionQuery, request.page(), request.pageSize());
+      WsUsersResponse WsUsersResponse = WsUsersResponse(permissionQuery, request.page(), request.pageSize());
 
-      writeProtobuf(usersResponse, wsRequest, wsResponse);
+      writeProtobuf(WsUsersResponse, wsRequest, wsResponse);
     } finally {
       dbClient.closeSession(dbSession);
     }
   }
 
-  private UsersResponse usersResponse(PermissionQuery permissionQuery, int page, int pageSize) {
+  private WsUsersResponse WsUsersResponse(PermissionQuery permissionQuery, int page, int pageSize) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)